package.json

```diff
@@ -5,8 +5,9 @@
     "ingest": "node update_remote_settings_records.mjs"
   },
   "devDependencies": {
-    "@mdn/browser-compat-data": "latest",
-    "btoa": "1.2.1",
-    "node-fetch": "^3.2.0"
+    "btoa": "1.2.1"
+  },
+  "dependencies": {
+    "node-fetch": "^3.2.10"
```

Comment:
nit: I think we can keep everything in `devDependencies`. If I remember correctly, dependencies are only relevant if you are publishing a package (ie dependencies are bundled in the package, devDependencies are not). It's not our case so we should be fine only with devDependencies.

Reply:
sounds about right, updated `package.json`